student.py

```diff
@@ -9,40 +12,43 @@ def __init__(self):
         self.students = {}
 
     def add_student(self, name, age, student_id):
-        if student_id in self.students:
+        # Error: Incorrect condition, should check if student_id in self.students
+        if student_id not in self.students:  
```

Comment:
The condition should check if `student_id` is in `self.students` to avoid adding a duplicate student.
```diff
- if student_id not in self.students:
+ if student_id in self.students:
```
---

Reply:
@sparklebot testing hello